pallets/crowdloan-rewards/src/lib.rs

```diff
@@ -317,9 +317,9 @@ pub mod pallet {
 	#[pallet::genesis_config]
 	pub struct GenesisConfig<T: Config> {
 		/// Contributions that have a native account id associated already.
-		pub associated: Vec<(T::RelayChainAccountId, T::AccountId, u32)>,
+		pub associated: Vec<(T::RelayChainAccountId, T::AccountId, BalanceOf<T>)>,
```

Comment:
These should not be BalanceOf<T>. That refers to our chain's native balance type. These are in relay balance units. If you want to use a nicer type than `u32` (which is totally reasonable), then it should be imported from Polkadot of Cumulus.